pkg/gui/controllers/helpers/search_helper.go

```diff
@@ -167,6 +174,31 @@ func (self *SearchHelper) CancelPrompt() error {
 	return self.c.PopContext()
 }
 
+func (self *SearchHelper) ScrollHistory(scrollIncrement int) {
+	state := self.searchState()
+
+	context, ok := state.Context.(types.IHistoryContext)
+	if !ok {
+		return
+	}
+	states := context.GetHistory()
+
+	if val, err := states.PeekAt(state.PrevSearchIndex + scrollIncrement); err == nil {
+		state.PrevSearchIndex += scrollIncrement
+
+		if _, ok := state.Context.(types.ISearchableContext); ok {
```

Comment:
is it necessary to set the search prefix here? I assumed that would already have been done

Reply:
could have sworn I needed it before. removed it since apparently I don't anymore.

8e73ffe7d5f43b0b51a293c0c3096f93d5966392